src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V3.cs

```diff
@@ -8,6 +8,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using FluentAssertions;
+using k8s.Models;
```

Comment:
what is this?

Reply:
Just a useless ns added automatically, will not be in the final PR for sure